### PR TITLE
[OSQuery] Add a known issue to the release notes for the 8.6.0 OSQuery Manager integration

### PR DESCRIPTION
**Pull request by kevinlog**
There is a known issue in the 8.6.0 Agent that we found today that we want to add to the release notes for the 8.6.0 release.

### CODE REVIEW

File: docs/release-notes/8.6.asciidoc [hunk 1]
```diff
@@ -8,7 +8,7 @@
 [discrete]
 [[known-issue-8.6.0]]
 ==== Known issues
-There are no known issues in 8.6.0.
+There is an issue with using the OSQuery Manager integration with the Agent which causes OSQuery results to not be properly written to Elasticsearch ahd, therefore, not viewable in Kibana (issue: https://github.com/elastic/beats/issues/34250).  It is recommended that OSQuery users not upgrade to 8.6.0 and wait for the 8.6.1 patch release.
```
Comment (caitlinbetz) [inline docs/release-notes/8.6.asciidoc:11]: very small, but "and" typo:

> ...to not be properly written to Elasticsearch **ahd**, therefore, not viewable

**Review by caitlinbetz — APPROVED** [review]

**Review by nastasha-solomon — COMMENTED** [review]
Thanks for filing this @kevinlog! I left some edits for your consideration.

File: docs/release-notes/8.6.asciidoc [hunk 2]
```diff
@@ -8,7 +8,7 @@
 [discrete]
 [[known-issue-8.6.0]]
 ==== Known issues
-There are no known issues in 8.6.0.
+There is an issue with using the OSQuery Manager integration with the Agent which causes OSQuery results to not be properly written to Elasticsearch and, therefore, not viewable in Kibana (issue: https://github.com/elastic/beats/issues/34250).  It is recommended that OSQuery users not upgrade to 8.6.0 and wait for the 8.6.1 patch release.
```
[the remark this answers is not in the document]
Comment (joepeeples) [reply docs/release-notes/8.6.asciidoc:11]: +1 to @nastasha-solomon's edit, except there looks like a typo in the last bit: "skip 8.6.1 and upgrade to 8.6.1 instead"

[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/release-notes/8.6.asciidoc:11]: fixed!

**Review by benironside — APPROVED** [review]
Nastasha's edit LGTM 👍

**Review by joepeeples — COMMENTED** [review]
LGTM with @nastasha-solomon's edit!

File: docs/release-notes/8.6.asciidoc (hunk 2 above)
[the remark this answers is not in the document]
Comment (kevinlog) [reply docs/release-notes/8.6.asciidoc]: @joepeeples @nastasha-solomon thank you for the suggestions!

I added one more phrase at the end "...when it's released".  My thinking was that it sounded like `8.6.1` was released already but it's not.  Happy to take it out if needed.  Here's what I pushed up.

```
When using the Osquery Manager integration with {agent}, Osquery results aren't properly written to {es} and therefore aren't viewable in Kibana ( https://github.com/elastic/beats/issues/34250)[#34250]. We recommend that Osquery users skip {stack} 8.6.0 and upgrade to {stack} 8.6.1 instead when it is released.
```

Let me know your thoughts.

[the remark this answers is not in the document]
Comment (joepeeples) [reply docs/release-notes/8.6.asciidoc]: Good point @kevinlog, and that reminds me that we should also probably say "8.6.1 _or later._" So...

` ...skip {stack} 8.6.0 and upgrade to (stack} 8.6.1 or later when available.`

**Review by nastasha-solomon — COMMENTED** [review]
Adding https://github.com/elastic/detection-rules/pull/2447

File: docs/release-notes/8.6.asciidoc [hunk 3]
```diff
@@ -8,7 +8,7 @@
 [discrete]
 [[known-issue-8.6.0]]
 ==== Known issues
-There are no known issues in 8.6.0.
+When using the Osquery Manager integration with {agent}, Osquery results aren't properly written to {es} and therefore aren't viewable in Kibana ( https://github.com/elastic/beats/issues/34250)[#34250]. We recommend that Osquery users skip {stack} 8.6.0 and upgrade to {stack} 8.6.1 instead when it is released.
```
[the remark this answers is not in the document]
Comment (approksiu) [reply docs/release-notes/8.6.asciidoc]: For 12 - the suggested fix will not work. Here's the suggested wording: Pre-built Detection rules investigation guides may not render correctly if they include an escaped character (such as `\"`). Updating the pre-built rules on the prompt will fix the issue. ( https://github.com/elastic/detection-rules/pull/2447[#2447]).

**Review by jmikell821 — COMMENTED** [review]
Slight fix - thanks!

File: docs/release-notes/8.6.asciidoc (hunk 3 above)
[the remark this answers is not in the document]
Comment (nastasha-solomon) [reply docs/release-notes/8.6.asciidoc]: Thanks for the feedback, @approksiu ! One question about the workaround - when users update prebuilt rules "on the prompt" are they performing this action in the prebuilt rules' settings or a different part of the rules UI?

[the remark this answers is not in the document]
Comment (approksiu) [reply docs/release-notes/8.6.asciidoc]: It's on the pre-built rules page - the prompt appear on top of the rules page.

**Review by jmikell821 — APPROVED** [review]
LGTM!

File: docs/release-notes/8.6.asciidoc [hunk 4]
```diff
@@ -8,7 +8,8 @@
 [discrete]
 [[known-issue-8.6.0]]
 ==== Known issues
-There are no known issues in 8.6.0.
+* When using the Osquery Manager integration with {agent}, Osquery results aren't properly written to {es} and, therefore, cannot be viewed in Kibana (https://github.com/elastic/beats/issues/34250)[#34250]). We recommend that Osquery users skip {stack} version 8.6.0 and upgrade to {stack} version 8.6.1 or later when available.
+* Osquery investigation guides may not render correctly if they include an escaped character (such as `\"`). To avoid breaking the query syntax, copy and paste affected queries directly into the investigation guide instead of importing them ( https://github.com/elastic/detection-rules/pull/2447[#2447]).
```
Comment (nastasha-solomon) [inline docs/release-notes/8.6.asciidoc:12]:
```suggestion
* Investigation guides for some prebuilt rules may not render correctly if they include an escaped character (such as `\"`). To resolve this, update your prebuilt rules once you receive a rule update prompt on the Rules page ( https://github.com/elastic/detection-rules/pull/2447[#2447]).
```

**Review by nastasha-solomon — COMMENTED** [review]
Revised summary to include Kseniia's input.

**Review by jmikell821 — APPROVED** [review]

**Pull request merged**